include/verilated_vcd_c.cpp

```diff
@@ -431,9 +431,9 @@ void VerilatedVcd::dumpHeader() {
 
             for (; *np && *np != ' ' && *np != '\t'; np++) {
                 if (*np == '[') {
-                    printStr("(");
+                    printStr("[");
                 } else if (*np == ']') {
-                    printStr(")");
+                    printStr("]");
```

Comment:
Suspect these code blocks can just be removed?

Reply:
Actually... I tried leaving those untouched and the VCD files still used () to index signals, not [ ]. I'm not sure why though...